### DISCUSSION TITLE
supabase client on browser

### DISCUSSION THREAD

**Discussion by asyarifstudio** [supabase/supabase#22155] [Questions] [answered]
Hi everyone

If we create a supabase client on browser side and perform database operation from browser, does it mean it will be limited to **Max Client Connections** in the database setting?

I was following the User Management tutorial with sveltekit and the sample create a supabase client on the browser side as well on the server side.  so I was wondering about this limitation 

thanks a lot for any help

**Comment by GaryAustin1** [comment, marked as answer]
No the REST API is independent from the database connection settings.  It will run as many REST requests as it can based on performance of your process/memory.  If too many requests queue up though it will eventually run out of connections to the database, but there is no direct connection between a browser and the REST API.
